### DISCUSSION TITLE
MPD stops running after Nov 14 update

### DISCUSSION THREAD

**Discussion by Battum** [rern/rAudio#624] [Bug Reports]
After the update of Nov 14 there are some problems:

When not connected to a BT soundbar selecting a source in the library shows up in the player and it plays. But after connecting to the soundbar (the dot is green) the MPD stops working. Only a reboot will solve that until ... I reconnect to the soundbar.
In the Player screen under Music Player Deamon there is this:

_● mpd 0.23.1-1
bind_to_address        "127.0.0.1"
db_file                "/srv/http/data/mpd/mpd.db"
music_directory        "/mnt/MPD"
playlist_directory     "/var/lib/mpd/playlists"
replaygain             "off"
state_file             "/srv/http/data/mpd/mpdstate"
user                   "mpd"

input {
	plugin         "curl"
}

decoder {
	plugin         "ffmpeg"
	enabled        "no"
}

resampler {
	plugin         "soxr"
	quality        "very high"
}

audio_output {
	name           "[AV] Samsung Soundbar - A2DP Playback Switc
[AV] Samsung Soundbar - A2DP Playback Volum"
	device         "bluealsa"
	type           "alsa"
	mixer_type     "software"
	format         "44100:16:2"
}

● mpd.service - Music Player Daemon
     Loaded: loaded (/usr/lib/systemd/system/mpd.service; disabled; vendor preset: disabled)
    Drop-In: /etc/systemd/system/mpd.service.d
             └─override.conf
     Active:  (Result: exit-code) since Wed 2021-11-17 12:10:48 CET; 2min 25s ago
       Docs: man:mpd(1)
             man:mpd.conf(5)
    Process: 1930 ExecStart=/usr/bin/taskset -c 3 /usr/bin/mpd --no-daemon (code=exited, status=1/FAILURE)
   Main PID: 1930 (code=exited, status=1/FAILURE)
        CPU: 339ms

Nov 17 12:10:48 rAudio systemd[1]: Starting Music Player Daemon...
Nov 17 12:10:48 rAudio taskset[1930]: exception: Error in /etc/mpd.conf line 24; Missing closing '"'
Nov 17 12:10:48 rAudio systemd[1]: mpd.service: Main process exited, code=exited, status=1/FAILURE
Nov 17 12:10:48 rAudio systemd[1]: mpd.service: Failed with result 'exit-code'.
Nov 17 12:10:48 rAudio systemd[1]: Failed to start Music Player Daemon._

Plugging in the HDMI cable and disabling BT gives a running MPD and sound.

**Comment by rern** [comment]
The name of Samsung Soundbar was split to 2 lines.
Try:
```sh
sed -i -e '/^\[AV\] Samsung Soundbar/d' -e 's/ Playback Switc/"/' /etc/mpd.conf
systemctl restart mpd
```
And post result of:
```sh
amixer -D bluealsa scontrols
```

**Comment by Battum** [comment]
Works!
output requested:

rAudio:~  #  amixer -D bluealsa scontrols
Simple mixer control '[AV] Samsung Soundbar - A2DP Playback Switc',0
Simple mixer control '[AV] Samsung Soundbar - A2DP Playback Volum',0

But after reboot changes are lost and problem reappears.
So what to change to make this permanent?

Second problem is that after reboot the bluetooth pairing is lost and has to be done over again. Is there a way to make the pairing permanent?

Peter
